src/main/webapp/app/exam/participate/summary/exam-participation-summary.component.ts

```diff
@@ -100,25 +146,15 @@ export class ExamParticipationSummaryComponent implements OnInit {
      * returns the students' submission for the exercise, undefined if no participation could be found
      */
     getSubmissionForExercise(exercise: Exercise) {
-        if (
-            exercise &&
-            exercise.studentParticipations &&
-            exercise.studentParticipations.length > 0 &&
-            exercise.studentParticipations[0].submissions &&
-            exercise.studentParticipations[0].submissions.length > 0
-        ) {
-            return exercise.studentParticipations[0].submissions[0];
-        }
+        return exercise?.studentParticipations?.[0]?.submissions?.[0];
```

Comment:
there was a reason for this extensive checking, please do not remove it, except some very experience developers agree that the code is 100% the same and we have extensive client tests that confirm this. 

Otherwise this might introduce issues

Reply:
Same as above. Code is equivalent and cleaner in my opinion, but I agree on the testing since this involves the exam component.
CC @bassner, our JS expert.

Reply:
Added unit tests for this simplification at commit https://github.com/ls1intum/Artemis/pull/4932/commits/9896b87f6e9769b29ac20b428dd0bfcf84dc56d2.